src/Components/Blazor/Blazor.Version.props

```diff
@@ -0,0 +1,8 @@
+<Project>
```

Comment:
Shouldn't need this file in 5.0. It might be easier to simply empty it's content to make future merges easier.